models/feature-flags.go

```diff
@@ -0,0 +1,12 @@
+package models
+
+type FeatureFlag struct {
+	Id          string `json:"id" dynamodbav:"Id"`
+	Name        string `json:"name" dynamodbav:"name"`
+	Description string `json:"description" dynamodbav:"description"`
+	CreatedAt   int64  `json:"createdAt" dynamodbav:"createdAt"` //was getting error on unmarshal ddb result with time.Time
```

Comment:
date and timestamps are to be stored in ISO 8601 or epoch format will int64 satisfy them?

Reply:
Yes `int64` can be used, dynamodb doesn't support `unmarshalling` to `time.Time` in Go